packages/strapi-generate-new/lib/resources/json/package.json.js

```diff
@@ -28,11 +28,11 @@ module.exports = opts => {
       lint: 'eslint api/**/*.js config/**/*.js plugins/**/*.js',
```

Comment:
can replace this by a simple `eslint .`
and add  to the eslintignore file plz :)
```txt
.cache
build
```

Reply:
Sure!